Documentation/ceph-quickstart.md

```diff
@@ -86,6 +86,35 @@ Before you start the operator in production, there are some settings that you ma
    2. Flex driver: The flex driver is deprecated in favor of the CSI driver, but can still be enabled with the `ROOK_ENABLE_FLEX_DRIVER` setting.
    3. Node affinity and tolerations: The CSI driver by default will run on any node in the cluster. To configure the CSI driver affinity, several settings are available.
 
+## Deploying to other namespaces
+
+If you wish to deploy the Rook Operator and/or Ceph clusters to alternate namespaces, the manifests
+are commented to allow for easy `sed` replacements. Change `ROOK_CLUSTER_NAMESPACE` to tailor the
+manifests for an additional Ceph cluster. You can choose to also change `ROOK_OPERATOR_NAMESPACE` to
+create a new Rook Operator for each Ceph cluster, or you can leave it at the same value for every
```

Comment:
If they are going to have one operator per cluster, they will also need to set [ROOK_CURRENT_NAMESPACE_ONLY: true](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/operator.yaml#L301). To be clear that this is a separate option, perhaps a separate paragraph?